ddtelemetry/src/data/payloads.rs

```diff
@@ -72,6 +72,11 @@ pub struct Log {
     pub level: LogLevel,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub stack_trace: Option<String>,
+    #[serde(skip_serializing_if = "String::is_empty")]
```

Comment:
Nice, I should use this too :)

Reply:
Actually I think I missed a matching `#[serde(default)]` annotation, because otherwise the field will be considered as missing on deserialization